packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import { checkIfColumnIsResponsive } from '../utils/utils';
+import columnShape from '../proptypes/columnShape';
+import { widthUnitTypes } from '../utils/constants';
+import styles from './Cell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * An identifier to uniquely identify the cell.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * Content that will be rendered within the Cell.
+   */
+  children: PropTypes.node,
+
+  /**
+   * A column data.
+   */
+  column: columnShape,
+
+  /**
+   * Columns minimum width in units set by widthUnit prop, such as `px`, `em`, or `rem`.
+   */
+  columnMinimumWidth: PropTypes.number,
+
+  /**
+   * Columns maximum width in units set by widthUnit prop, such as `px`, `em`, or `rem`.
+   */
+  columnMaximumWidth: PropTypes.number,
+
+  /**
+   * The type of width value. One of `px`, `em`, `rem`. Defaults to 'px'.
+   */
+  widthUnit: PropTypes.oneOf([
+    widthUnitTypes.PX,
+    widthUnitTypes.EM,
+    widthUnitTypes.REM,
+  ]).isRequired,
+};
+
+const Cell = (props) => {
+  const {
+    id,
+    children,
+    column,
+    columnMinimumWidth,
+    columnMaximumWidth,
+    widthUnit,
+  } = props;
+
+  const theme = useContext(ThemeContext);
+  const className = cx('cell', theme.className);
+
+  const {
+    width,
+    flexGrow,
+    alignToCenter,
+    maximumWidth,
+    minimumWidth,
+  } = column;
+
+  const isResponsive = checkIfColumnIsResponsive(flexGrow, width);
+
+  const style = {
+    flex: isResponsive ? `1 1 ${width || columnMinimumWidth}${widthUnit}` : null,
+    width: isResponsive ? null : `${width}${widthUnit}`,
+    justifyContent: `${alignToCenter ? 'center' : 'left'}`,
+    maxWidth: isResponsive && `${Math.min(maximumWidth, columnMaximumWidth)}${widthUnit}`,
+    minWidth: isResponsive && `${Math.max(minimumWidth, columnMinimumWidth)}${widthUnit}`,
+  };
+
+  return (
+    <div
+      id={id}
```

Comment:
Does your Cell component really need to have an id attribute?  Your current implementation does not guarantee uniqueness and could lead to an invalid DOM.

Reply:
Removed in https://github.com/cerner/terra-framework/commit/ae408c7c30b997ee75cf380978e4fdc763c5a240